apps/signal/51-function-call-effect/src/app/action.component.ts

```diff
@@ -39,7 +40,8 @@ export class ActionsComponent {
 
   constructor() {
     effect(() => {
-      this.userService.log(this.action() ?? 'No action selected');
+      const name = untracked(() => this.userService.name());
+      console.log(`${name}: ${this.action() ?? 'No action selected'}`);
```

Comment:
that's still not good. I think I need to change the challenge. You cannot access name and you still need to call the log function. 😅

Reply:
I thought about it a bit more and had another idea.

Third time's a charm?